media/pages/sk/vybrane-pripady/page.html

```diff
@@ -0,0 +1,4 @@
+{# vim: set filetype=htmldjango shiftwidth=2 :#}
+
+{% include "main/homepage/casestudy/main.html" %}
```

Comment:
@martinmacko47 Ak (ne)chceme iny heading alebo ho zarovnat dolava musime dat `<h2>` tag mimo pomocny template `casestudy/main.html`

Reply:
Takto to importovat nie je dobre z viacerych dovodov:
- Spominany nadpis
- Jednotlive pripady chceme usporiadat podla datumu. Takze blogy a casestudies mozu byt premiesane
- Casom budeme mat viac blogov/pripadov nez len 2/3. Potom na home page zobrazime prvych niekolko a tu vsetky. 
- Blogy su naformatovane inac. Tu v tomto zozname ich chceme naformatovat rovnako ako case studies. Lebo teraz to vyzera divne

Preto jednotlive pripady/bloky treba importovat po jednom. Usporiadat podla datumu pripady aj bloky spolu. A pridat nadpis tak, aby obrazovka vyzerala ako ostatne pages obrazovky.

Nakoniec byde treba vymysliet, co s blogmi, aby v tomto zozname vyzerali rovnako. Asi najrozumnejsie bude ich skopirovat a spravit z nich aj case studies. Akurat tieto case studies sa nezobrazia na homepage, ale iba tu.

Pripadne ak to pojde mozeme blog templaty spravit univerzalne a v hlavicke namiesto
```
{% extends "main/homepage/blogs/base.html" %}
```
pouzit
```
{% extends base_template|default:"main/homepage/blogs/base.html" %}
```
Potom pri includnuti s overridnutym `base_template` sa to vyrendruje ako casestudy.
```
{% include "main/homepage/blogs/sk/rozdiely_vo_vybavovani.html" with base_template="main/homepage/casestudy/base.html" %}
```

Reply:
:ok: